protoc_plugin/test/goldens/topLevelEnum.pbjson

```diff
@@ -5,27 +5,27 @@
 // @dart = 2.12
 
 // ignore_for_file: annotate_overrides, camel_case_types
-// ignore_for_file: constant_identifier_names
+// ignore_for_file: constant_identifier_names, deprecated_member_use
 // ignore_for_file: deprecated_member_use_from_same_package, directives_ordering
 // ignore_for_file: library_prefixes, non_constant_identifier_names
 // ignore_for_file: prefer_final_fields, return_of_invalid_type
-// ignore_for_file: unnecessary_const, unnecessary_import, unnecessary_this
-// ignore_for_file: unused_import, unused_shown_name
+// ignore_for_file: unnecessary_import, unnecessary_this, unused_import
+// ignore_for_file: unused_shown_name
 
 import 'dart:convert' as $convert;
 import 'dart:core' as $core;
 import 'dart:typed_data' as $typed_data;
 @$core.Deprecated('Use phoneTypeDescriptor instead')
-const PhoneType$json = const {
+const PhoneType$json = {
   '1': 'PhoneType',
-  '2': const [
-    const {'1': 'MOBILE', '2': 0},
-    const {'1': 'HOME', '2': 1},
-    const {'1': 'WORK', '2': 2},
-    const {'1': 'BUSINESS', '2': 2},
+  '2': [
+    {'1': 'MOBILE', '2': 0},
+    {'1': 'HOME', '2': 1},
+    {'1': 'WORK', '2': 2},
+    {'1': 'BUSINESS', '2': 2},
   ],
 };
 
 /// Descriptor for `PhoneType`. Decode as a `google.protobuf.EnumDescriptorProto`.
-final $typed_data.Uint8List phoneTypeDescriptor = $convert.base64Decode('CglQaG9uZVR5cGUSCgoGTU9CSUxFEAASCAoESE9NRRABEggKBFdPUksQAhIMCghCVVNJTkVTUxAC');
+final $typed_data.Uint8List phoneTypeDescriptor = $convert.base64Decode('CglQaG9uZVR5cGUSCgoGTU9CSUxFEAASCAoESE9NRRABEggKBFdPUksQAhIMCghCVVNJTkVTUx''AC');
```

Comment:
This is longer than 80 characters, is this expected? Is this a problem with the formatter?

I think the intended formatting is something like

```dart
final $typed_data.Uint8List phoneTypeDescriptor = $convert.base64Decode(
  'CglQaG9uZVR5cGUSCgoGTU9CSUxFEAASCAoESE9NRRABEggKBFdPUksQAhIMCghCVVNJTkVTUx'
  'AC');
```

Reply:
Ah, good catch.

I adjusted the output a bit so this will look better w/o formatting, but I think what we're seeing here is that these `test/goldens/*.pbjson` files aren't getting the formatter run on them after generation. (separately, I don't know why they don't end in a .dart extension; they do seem to contain dart code and not json)